domaci22.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+<link rel="stylesheet" href="stil22.css"> 
+
+</head>
+
+<body>  <div class= "container">
+    
+    
+      <div class = "split">
+        <h1 class="grad"> Život u Splitu </h1>  
+        <p class="caremipsum"> Carem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor <br> incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud <br> 
+            exercitation ullamcu laboris nisi ut aliquip ex ea commode consequat. Aed do.
+        </p>
+        <br>
+        <button class="botun" id="zuta" type="button"  > Građani </button>
```

Comment:
There is not need for id on this one, also, try to write code on english, so `.botun` becomes `.button`